yarn.lock

```diff
@@ -6146,14 +6146,6 @@ chainsaw@~0.1.0:
   dependencies:
     traverse ">=0.3.0 <0.4"
 
-chalk@^4.0.0:
```

Comment:
Not sure why this changed. If the CI build fails you may have to manually revert it. (`git checkout master -- yarn.lock`)